src/User.cpp

```diff
@@ -22,6 +26,8 @@ std::string User::getAllInfo() const
     ss<<std::setw(1) << "3." << group_ << "|";
     ss<<std::setw(1) << "4." << gitHub_ << "|";
     ss<<std::setw(1) << "5." << firecode_ << "|";
+    ss<<std::setw(1) << "6." << email_ << "|";
+    ss<<std::setw(1) << "7." << password_ << "|";
```

Comment:
printing password is not a good idea

Reply:
Corrected, Line with password has been removed.